### PR TITLE
Add RouteGroup support for kube-metrics-adapter

### PR DESCRIPTION
**Pull request by mikkeloscar**
Add support for RouteGroup based scaling in HPAs

Ref: https://github.com/zalando-incubator/kube-metrics-adapter/pull/349

### CODE REVIEW

File: test/e2e/kube_metrics_adapter_test.go [hunk 1]
```diff
@@ -169,6 +217,21 @@ func (tc *CustomMetricTestCase) Run() {
 		Expect(err).NotTo(HaveOccurred())
 
 	}
+
+	// check if a RouteGroup needs to be created
+	if tc.routegroup != nil {
+		// Create a Service for the Ingress
```
Comment (jonathanbeber) [inline test/e2e/kube_metrics_adapter_test.go]:
```suggestion
		// Create a Service for the RouteGroup
```

**Review by jonathanbeber — COMMENTED** [review]
:nail_care:

**Comment by jonathanbeber** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**